pkg/ccl/crosscluster/replicationutils/utils.go

```diff
@@ -237,6 +239,25 @@ func InvestigateFingerprints(
 	return nil
 }
 
+func GetDbAndSchemaPrefixesForTable(
+	ctx context.Context, txn *kv.Txn, descriptors *descs.Collection, tableDesc descpb.TableDescriptor,
+) (string, error) {
+	var prefixes string
+	byIDGetter := descriptors.ByID(txn).WithoutNonPublic().Get()
+	dbDesc, err := byIDGetter.Database(ctx, tableDesc.GetParentID())
+	if err != nil {
+		return "", err
+	}
+
+	scDesc, err := byIDGetter.Schema(ctx, tableDesc.GetUnexposedParentSchemaID())
```

Comment:
i think it's recomended to use `GetParentSchemaID()`. see note [here](https://github.com/msbutler/cockroach/blob/butler-skip-kms-roachtest/pkg/sql/catalog/descpb/structured.proto#L885)